src/geometries/PlaneGeometry.js

```diff
@@ -46,9 +48,10 @@ class PlaneGeometry extends BufferGeometry {
 				vertices.push( x, - y, 0 );
 
 				normals.push( 0, 0, 1 );
-
-				uvs.push( ix / gridX );
-				uvs.push( 1 - ( iy / gridY ) );
+				
+				//Use pre-computed inverse values
+				uvs.push(ix * invGridX);
+				uvs.push(1 - (iy * invGridY);
```

Comment:
Imo, this kind of performance optimization is too specific and I doubt you can measure a noticeable difference in browser environments. The previous version was also a bit easier to read.